packages/eslint-plugin/src/utils/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { AST_NODE_TYPES, ASTUtils, TSESTree } from '@typescript-eslint/utils';
```

Comment:
is it necessary to use typescript-eslint to create the lint rule ?

Reply:
AFAIR there are no similar utils in ESLint itself. I was using [eslint-plugin-testing-library](https://github.com/testing-library/eslint-plugin-testing-library) as a reference.